pkg/operator/staticpod/controllers.go

```diff
@@ -33,15 +33,34 @@ type staticPodOperatorControllers struct {
 // 4. BackingResourceController - this creates the backing resources needed for the operand, such as cluster rolebindings and installer service
 //    account.
 // 5. MonitoringResourceController - this creates the service monitor used by prometheus to scrape metrics.
-func NewControllers(targetNamespaceName, staticPodName string, command, revisionConfigMaps, revisionSecrets []string,
+func NewControllers(targetNamespaceName, staticPodName string, command, revisionConfigMaps, optionalRevisionConfigMaps, revisionSecrets []string,
```

Comment:
why isn't this the same revision resource?  I expect that ultimately everything that took the list of resources needs to be updated to handle the idea of an optional resource.  The compile breaks will help you.

Reply:
fixed this.